packages/world/src/ResourceSelector.sol

```diff
@@ -3,53 +3,65 @@ pragma solidity >=0.8.0;
 import { ROOT_NAMESPACE, ROOT_NAME } from "./constants.sol";
 import { Bytes } from "@latticexyz/store/src/Bytes.sol";
 
-bytes16 constant ROOT_NAMESPACE_STRING = bytes16("ROOT_NAMESPACE");
-bytes16 constant ROOT_NAME_STRING = bytes16("ROOT_NAME");
-
+/**
+ * A ResourceSelector is a 32-byte value that uniquely identifies a resource
+ * - 2 bytes for the resource type
+ * - 14 bytes for the namespace
```

Comment:
decided to steal from the namespace rather than name, but we could also steal 1 byte each (`bytes2`+`bytes15`+`bytes15`)

Reply:
> feels right! agree with stealing from the namespace (I actually like two different types bytes14 and bytes16 to distinguish them better)